### DISCUSSION TITLE
안녕하세요 IWP 관련해서 질문이 있습니다.

### DISCUSSION THREAD

**Discussion by changhyeok-an** [naverz/zepeto-studio-kor#1274] [5. Q&A]
안녕하세요 IWP 관련해서 질문이 있습니다. IWP 가이드를 보는데로 다 넣어서 가이드 대로 했는데도 아이템을 사용할 수 없습니다 라고 뜨고 있습니다 유니티통해서 하고 있는데 혹시 이런 경우에는 어떻게 해결 방법이 없는지 궁금합니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, '아이템을 사용할 수 없습니다'라고 뜨는 원인은 크게 두 가지입니다.

**1.제페토 관리자 측에서 부적절한 아이템으로 판단하여 비활성화 했을 경우 :** 
보통 출시 월드에서 문제가 생겼을 때 조치를 취하는 것이기에 이 경우는 아닐 것입니다.
활성화 여부는 제페토 스튜디오에서 확인이 가능합니다.

**2.IWP 구매 완료를 클라이언트와 서버 양쪽에서 연동하였을 경우 :** 
클라이언트와 서버 양쪽에 모두 구매 완료 이벤트를 엮을 경우 정상적인 구매가 불가해집니다. 
따라서 구현하시려는 월드가 멀티플레이일 경우에는 클라이언트에서는 구매 연동 스크립트를 삭제 부탁 드리며
서버 코드로만 구현해 주세요.

아래 가이드 및 영상을 참고하여 주세요.
https://studio.zepeto.me/guides/handling-iwp-event(server)
https://youtu.be/d-MaqjP-vjA

감사합니다.

**Comment by 17beans** [comment]
월드 커버는 등록하셨을까요? 월드 커버 이미지를 등록하셔야 IWP 기능을 정상적으로 사용 가능합니다.
